src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java

```diff
@@ -66,6 +66,34 @@
  * <pre>
  * &lt;module name=&quot;MutableException&quot;/&gt;
```

Comment:
Please add two more configs and examples for each property with a custom value: `format` and `extendedClassNameFormat`